xdata/xodtemplate.h

```diff
@@ -828,7 +828,6 @@ int xodtemplate_resolve_hostescalation(xodtemplate_hostescalation *);
 int xodtemplate_resolve_hostextinfo(xodtemplate_hostextinfo *);
 int xodtemplate_resolve_serviceextinfo(xodtemplate_serviceextinfo *);
 
-int xodtemplate_merge_extinfo_ojects(void);
```

Comment:
My default correction would be `objects`, but the item appears unused, so instead I've just removed it...